### PR TITLE
Added standard performance indicators for HX

### PR DESCRIPTION
**Pull request by nabily-met**
## Goal

Added standard performance indicators to heat exchangers.

Feedwater heaters:
- `FTR` Feedwater Temperature Rise. Difference between the feedwater inlet and outlet temperatures
- `TTD` Terminal Temperature Difference. Difference between hot side saturation temperature and feedwater outlet temperature
- `DCA` Drain Cooler Approach. Difference between hot side drain temperature and feedwater inlet temperature

Economizer:
Only `FTR` since the hot fluid is flue gases

Superheaters:
- `DT_superheat`: Outlet temperature difference to saturation (degrees of superheat)
- `STR` Steam Temperature Rise (only for the multifluid superheater): Steam temperature difference across the HX
- `TTD` and `DCA`: Only for water/water superheater

(No modifications are needed in the Changelog)

## Type of change 

- [ ] Bugfix
- [x] New feature
- [ ] Refactoring change
- [ ] Release & Version Update (don't forget to change the version number in `package.mo`)

Will it break anything in previous models ?

- [ ] Breaking change (If yes, make sure to point it out in the changelog)
- [x] Non-Breaking change

## Checklist

- [x] I have added the appropriate tags, reviewers, projects (and detailed the size and priority of my PR) and linked issues to this PR
- [x] I have performed a self-review of my own code
- [x] I have checked that all existing tests pass.
- [x] I have added/updated tests that prove my development works and does not break anything.
- [ ] I have made corresponding changes or additions to the documentation (in [Notion documentation](https://www.notion.so/metroscope/Metroscope-Modeling-Library-Documentation-MML3-WIP-50c8703c294446059d3b4a70d6ae4a71))
- [x] I have added corresponding entries to the [Changelog](../CHANGELOG.md)
- [x] I have checked for conflicts with target branch, and merged/rebased in consequence

You can also fill these out after creating the PR, but make sure to **check them all before submitting your PR for review**.

### CODE REVIEW

**Review by pierre-eliep-met — APPROVED** [review]
All good for me, maybe wait for @valentind-met 's review before merging ?

File: MetroscopeModelingLibrary/MultiFluid/HeatExchangers/Evaporator.mo
Comment (valentind-met) [inline MetroscopeModelingLibrary/MultiFluid/HeatExchangers/Evaporator.mo:35]: Small typo in the comment "colde"

File: MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo
Comment (valentind-met) [inline MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo:129]: Wouldn't a zero pinch fail the simulation anyway? Not sure this assert is relevant. Maybe flagging low pinches (<1°C ?) would be more interesting.

**Review by valentind-met — APPROVED** [review]

File: MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo
Comment (nabily-met) [reply MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo:129]: It simulates, I tested it. Because sometimes the energy balance is correct if we have sufficient flow.

Comment (pierre-eliep-met) [reply MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo:129]: agreed !

Comment (valentind-met) [reply MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo:129]: Maybe in this case let's add a second assert at the info level, for low pinches

Comment (nabily-met) [reply MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo:129]: I didn't understand the. For flue gases heat exchangers, there's an assert for W < 0, do you mean something similar ?

Comment (valentind-met) [reply MetroscopeModelingLibrary/MultiFluid/HeatExchangers/FuelHeater.mo:129]: No, I meant an assert for low pinches (not zero pinches), like pinch < 1°C

**Pull request merged**
